### ISSUE TITLE
7.1.2.155 Exception in server tick loop

### ISSUE THREAD

**Issue by InsaneJ** [aidancbrady/Mekanism#2095]
Using Mekanism 7.1.2.155 and Fastcraft 1.16.
Someone placed a Mekanism generator in flowing lava on our server which produced the following error:
```
java.lang.NoSuchFieldError: enableAmbientLighting
	at mekanism.generators.common.block.BlockGenerator.getLightValue(BlockGenerator.java:139)
	at fastcraft.a.e.a(Unknown Source)
	at fastcraft.a.e.a(Unknown Source)
	at fastcraft.a.e.a(Unknown Source)
	at fastcraft.d.a(Unknown Source)
	at fastcraft.Hooks.d(Unknown Source)
	at net.minecraft.world.World.func_147465_d(World.java:460)
	at net.minecraft.item.ItemBlock.placeBlockAt(ItemBlock.java:206)
	at mekanism.generators.common.item.ItemBlockGenerator.placeBlockAt(ItemBlockGenerator.java:166)
	at net.minecraft.item.ItemBlock.func_77648_a(ItemBlock.java:103)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```

Crash-report: http://paste2.org/4msJKcEX

Not sure if it's a Mekanism or Fastcraft issue so posting it here just in case it might be useful. Also posted on IC2 forum here: http://forum.industrial-craft.net/index.php?page=Thread&postID=178897#post178897

**Comment by halvors** [comment]
This is fixed in the latest build, try out 7.1.2.157. That should fix your problem :-)

**Comment by InsaneJ** [comment]
Thanks!
